From 928e8646b8441bd5fc8dc5882f493e2338e2a5aa Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Fri, 23 Dec 2022 09:29:22 +0100
Subject: [PATCH] Support cross-compilation with sysroot directory

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 6245dbc..d1b2bc7 100644
--- a/meson.build
+++ b/meson.build
@@ -61,7 +61,7 @@ if libdrm.version().version_compare('<2.4.113')
 elif libdrm.type_name() == 'internal'
   fourcc_h = meson.current_source_dir() / 'subprojects/libdrm/include/drm/drm_fourcc.h'
 else
-  fourcc_h = libdrm.get_pkgconfig_variable('includedir') / 'libdrm/drm_fourcc.h'
+  fourcc_h = libdrm.get_pkgconfig_variable('pc_sysrootdir') + libdrm.get_pkgconfig_variable('includedir') / 'libdrm/drm_fourcc.h'
 endif
 
 if libpci.found()
-- 
2.37.2

